### ISSUE TITLE
App crashes after trying to bind Pause key to playback control

### ISSUE THREAD

**Issue by valipod** [ytmdesktop/ytmdesktop#1175]
### Consent

- [X] I verified that there is no open/closed issue for the same subject.
- [X] I understand that **YTMDesktop have NO affiliation with Google or YouTube**

### Current Behavior

Unaware of #1171 (this is not a duplicate), I tried binding the keyboard Pause to the play/pause action. At that moment the app crashed:
```
YouTube Music Desktop App Crashed

Environment Details:
    YouTube Music Desktop App 2.0.0-rc.3
    Electron 26.2.1
    Windows NT 10.0.22000

Name: TypeError
Message: Error processing argument at index 0, conversion failure from Pause
Cause: Unknown

TypeError: Error processing argument at index 0, conversion failure from Pause
    at B (C:\Users\Valentin\AppData\Local\youtube_music_desktop_app\app-2.0.0-rc3\resources\app\.webpack\main\index.js:8:675725)
    at App.<anonymous> (C:\Users\Valentin\AppData\Local\youtube_music_desktop_app\app-2.0.0-rc3\resources\app\.webpack\main\index.js:8:688822)
    at App.emit (node:events:513:28)
```

and I am unable to start it anymore. I have tried with uninstall/reinstall, but the problem remains, the app crashes on start. After uninstall, the folder `\AppData\Local\youtube_music_desktop_app` is completely deleted, so no local configuration remains there before reinstalling.

### Expected Behavior

In the context of #1171, the expected behavior would be that the app at least doesn't crash :)

### Steps To Reproduce

1. Install the app (2.0.0.rc3, Windows 11)
2. In settings, try to assign the keyboard Pause key to the play/pause action.

### YTMDesktop

2.0.0-rc.3

### OS

Windows

### OS Version

11

### Arch

x64

### Installation way

YouTube.Music.Desktop.App-2.0.0-rc.3.Setup.exe

**Comment by NovusTheory** [comment]
Closing as this is part of rc.4 release

**Comment by valipod** [comment]
Please advise how to 'uncrash' the current situation, installing the RC4 doesn't help:

`YouTube Music Desktop App Crashed

Environment Details:
    YouTube Music Desktop App 2.0.0-rc.4
    Electron 26.2.1
    Windows NT 10.0.22000

Name: TypeError
Message: Error processing argument at index 0, conversion failure from Pause
Cause: Unknown

TypeError: Error processing argument at index 0, conversion failure from Pause
    at H (C:\Users\Valentin\AppData\Local\youtube_music_desktop_app\app-2.0.0-rc4\resources\app\.webpack\main\index.js:8:496404)
    at App.<anonymous> (C:\Users\Valentin\AppData\Local\youtube_music_desktop_app\app-2.0.0-rc4\resources\app\.webpack\main\index.js:8:516817)
    at App.emit (node:events:513:28)`

**Comment by valipod** [comment]
:) actually a better and simple approach should be to delete the whole folder.

I was under the impression that the installation folder is in `\AppData\Local\youtube_music_desktop_app`, so I kept deleting that folder, and that was going nowhere, but now that you mention `\AppData\Roaming\YouTube Music Desktop App`, I noticed there three folders with different naming schemes (probably leftovers from older versions), so after deleting them, I just had to re-authenticate and that was that.

Thank you.
